user_sync/rules.py

```diff
@@ -126,6 +142,7 @@ def read_desired_user_groups(self, mappings, directory_connector):
         directory_group_filter = options['directory_group_filter']
         if (directory_group_filter != None):
             directory_group_filter = set(directory_group_filter)
+        extended_attributes = options.get('extended_attributes')
```

Comment:
See comment above at line 317.  We need to make sure these don't overlap with the base attributes.